src/googmodule.ts

```diff
@@ -209,17 +209,29 @@ function rewriteModuleExportsAssignment(expr: ts.ExpressionStatement) {
  * @return An array of statements if it converted, or null otherwise.
  */
 function rewriteCommaExpressions(expr: ts.ExpressionStatement): ts.Statement[]|null {
-  // Early exit if the outer statement isn't a comma statement.
-  if (!ts.isBinaryExpression(expr.expression)) return null;
-  if (expr.expression.operatorToken.kind !== ts.SyntaxKind.CommaToken) return null;
+  if (expr.expression.kind === ts.SyntaxKind.CommaListExpression ||
+      (ts.isBinaryExpression(expr.expression) &&
+       expr.expression.operatorToken.kind === ts.SyntaxKind.CommaToken)) {
+    return visit(expr.expression);
+  }
+
+  return null;
 
   // Recursively visit comma-separated subexpressions, and collect them all as
   // separate expression statements.
-  return visit(expr.expression);
   function visit(expr: ts.Expression): ts.Statement[] {
+    // There are two representation for comma expressions:
+    // 1) a tree of  "binary expressions" whose contents are comma operators
     if (ts.isBinaryExpression(expr) && expr.operatorToken.kind === ts.SyntaxKind.CommaToken) {
       return visit(expr.left).concat(visit(expr.right));
     }
+    // or,
+    // 2) a "comma list" expression, where the subexpressions are in one array
+    if (expr.kind === ts.SyntaxKind.CommaListExpression) {
+      const commaList = expr as ts.CommaListExpression;
+      // TODO(blickly): Simplify using flatMap once node 11 available
+      return commaList.elements.reduce((acc: ts.Statement[], x) => acc.concat(visit(x)), []);
```

Comment:
I guess this does N separate concats.   You might instead pass all the subarrays to concat, via something like:

`[].concat(elements.map(stuff)...)`

I don't think it matters too much either way though.

Reply:
Good point. That way is certainly simpler.